TYPO3.Neos/Configuration/Settings.yaml

```diff
@@ -161,6 +161,33 @@ TYPO3:
         'ru': 'Pусский – Russian'
 #        'sv': 'Svenska – Swedish'
 
+      # Aloha presets used in nodeType configuration
+      aloha:
+        presets:
+          'Standard':
```

Comment:
+1 lowercase and default

What about the other configs? Like for Headlines and so on?